samples/bot-configuration-app/nodejs/appManifest/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
```

Comment:
Please check if we need to push it with DevPreview only.

Reply:
Yes , It is mentioned in document as well.

![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/140516784/53289b8c-8e19-4644-ac8e-ec17b7d6f344)